reference/service/orgs.md

```diff
@@ -18,12 +18,53 @@ Adding a new Pulumi organization can be done directly from the Pulumi Cloud Cons
     <button class="button">ADD ORGANIZATION</button>
 </a>
 
-## GitHub-backed Organizations
+## GitHub and GitLab-backed Organizations
 
-Organizations using the Pulumi Team and Enterprise editions may be backed by a GitHub organization,
-enabling you to manage user access and team permissions by using your existing GitHub access
+Organizations using the Pulumi Team and Enterprise editions may be backed by a GitHub organization or GitLab group,
```

Comment:
Again, I'd just drive users to the table below, since that is the most efficient way we'll communicate the types of organizations we support. And also how Teams would map.

Perhaps:

"""Organizations using the Pulumi Team and Enterprise editions may be backed by an external service, such as GitHub or GitLab. You can enforce user access and team permissions in the Pulumi organization by using your existing access controls on another service.

Here's a table describing the mapping between supported 3rd party services and how they are mirrored into Pulumi:"""

Does it make sense to add another row with "Team" as well? Which IIRC in the GitLab case is "Sub-group"?